qemu/tests/cfg/chardev_backends.cfg

```diff
@@ -0,0 +1,9 @@
+- chardev_backends: 
```

Comment:
`chardev_backend` have special meaning, it is not suitable to use `chardev_backends` as an case name, how do yo think about `virtio_serial_throttling` ?  @nanliu-r @michiboo